### PR TITLE
Add initial version of the app's front-end

### PR DESCRIPTION
**Pull request by jytug**
Currently it's burping Kaczmarski so not much but we'll get there

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+To prepare the env, run
+```
+python3 -m venv venv
+source ./venv/bin/activate.py
+python -m pip install -r requirements.txt
+```
+
+Running the app
+===============
+To run the amazing Kaczmarskify app, run
```
Comment (sapkos) [inline README.md]: Ka**r**czmarskify

**Comment by ronich** [comment]
pls add `__pycache__` to `.gitignore`

File: app/animate.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    @window.event
+    def on_draw():
+        window.clear()
+        state['image'].blit(0, 0)
+
+    @window.event
+    def on_mouse_press(x, y, button, modifiers):
+        if button == mouse.LEFT:
+            state['image'] = face_open
+            burp.play()
```
Comment (ronich) [inline app/animate.py]: this raises exceptions when one is clicking uncontrollably in anticipation of the next burp

```Traceback (most recent call last):
  File "_ctypes/callbacks.c", line 234, in 'calling callback function'
  File "karczmarskify/venv/lib/python3.6/site-packages/pyglet/libs/darwin/cocoapy/runtime.py", line 1121, in objc_method
    result = f(py_self, *args)
  File "karczmarskify/venv/lib/python3.6/site-packages/pyglet/window/cocoa/pyglet_view.py", line 291, in mouseDown_
    self._window.dispatch_event('on_mouse_press', x, y, buttons, modifiers)
  File "karczmarskify/venv/lib/python3.6/site-packages/pyglet/window/__init__.py", line 1330, in dispatch_event
    if EventDispatcher.dispatch_event(self, *args) != False:
  File "karczmarskify/venv/lib/python3.6/site-packages/pyglet/event.py", line 416, in dispatch_event
    if handler(*args):
  File "karczmarskify/app/animate.py", line 27, in on_mouse_press
    burp.play()
  File "karczmarskify/venv/lib/python3.6/site-packages/pyglet/media/codecs/base.py", line 283, in play
    player.queue(self)
  File "karczmarskify/venv/lib/python3.6/site-packages/pyglet/media/player.py", line 193, in queue
    self._source = source.get_queue_source()
  File "karczmarskify/venv/lib/python3.6/site-packages/pyglet/media/codecs/base.py", line 414, in get_queue_source
    raise MediaException('This source is already a source on a player.')
pyglet.media.exceptions.MediaException: This source is already a source on a player.```

Comment (jytug) [reply app/animate.py]: This is resolved now please approve!!!!!!!!!! (@ronich @sapkos )

File: app/animate.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    @window.event
+    def on_mouse_press(x, y, button, modifiers):
+        if button == mouse.LEFT:
+            state['image'] = face_open
+            if not burp.is_player_source:
+                burp.play()
+
+    @window.event
+    def on_mouse_release(x, y, button, modifiers):
+        global curr_image
```
Comment (radogost) [inline app/animate.py]: Do you need this line?

**Comment by ronich** [comment]
this is outstanding work!

**Pull request merged**